src/BasicViewer.js

```diff
@@ -33,17 +35,32 @@ var BasicViewer = function(catTitle, features, container, fv) {
                         return 'up_pftv_feature up_pftv_' + d.type.toLowerCase();
                     })
                 ;
+                newShapes
```

Comment:
newShapes corresponds to the enter selection, so it should not need the exit. Changed one of the shapes to newShapes because newShapes is the one that has the new shapes and we need to update color and path for all of them